notation.go

```diff
@@ -327,7 +327,7 @@ func Verify(ctx context.Context, verifier Verifier, repo registry.Repository, ve
 	}
 
 	var verificationOutcomes []*VerificationOutcome
-	errExceededMaxVerificationLimit := ErrorVerificationFailed{Msg: fmt.Sprintf("total number of signatures associated with an artifact should be at most: %d", verifyOpts.MaxSignatureAttempts)}
+	errExceededMaxVerificationLimit := ErrorVerificationFailed{Msg: fmt.Sprintf("the number of signatures has surpassed the maximum limit of %d that can be evaluated", verifyOpts.MaxSignatureAttempts)}
```

Comment:
```suggestion
	errExceededMaxVerificationLimit := ErrorVerificationFailed{Msg: fmt.Sprintf("signature evaluation stopped because the configured limit of %d signatures to verify per artifact exceeded", verifyOpts.MaxSignatureAttempts)}
```

Reply:
Here is my proposal: `signature evaluation stopped. The configured limit of %d signatures to verify per artifact exceeded`

Reply:
Agree. Code updated.